doc/Command-Reference.md

```diff
@@ -6564,6 +6568,301 @@ This command is used to set the counter polling interval. Default is 20 seconds.
 
 Go Back To [Beginning of the document](#) or [Beginning of this section](#sflow)
 
+## SNMP
+
+### SNMP Show commands
+
+**show runningconfiguration snmp**
+
+This command displays the global SNMP configuration that includes the location, contact, community, and user setttings.
+
+- Usage:
+  ```
+  show runningconfiguration snmp
+  ```
+
+- Example:
+  ```
+  admin@switch1:~$ show runningconfiguration snmp 
+  Location
+  ------------
+  Emerald City
+
+
+  SNMP_CONTACT    SNMP_CONTACT_EMAIL
+  --------------  --------------------
+  joe             joe@contoso.com
+
+
+  Community String    Community Type
+  ------------------  ----------------
+  Jack                RW
+
+
+  User    Permission Type    Type    Auth Type    Auth Password    Encryption Type    Encryption Password
+  ------  -----------------  ------  -----------  ---------------  -----------------  ---------------------
+  Travis  RO                 Priv    SHA          TravisAuthPass   AES                TravisEncryptPass
+  admin@switch1:~$ 
```

Comment:
Suggest deleting the trailing prompt, it's unnecessary and just makes the document longer. Please do throughout wherever one appears.

Reply:
fixed.